elastic-utils-lib/resources/docker/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.5.2
+    container_name: cmr-es-elastic
+    environment:
+      - http.host=0.0.0.0
+      - transport.host=127.0.0.1
+      - ELASTIC_PASSWORD=kibana
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms1024m -Xmx1024m"
+      - xpack.security.enabled=false
+      - xpack.monitoring.enabled=false
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    volumes:
+      - ${PWD}/data/elastic:/usr/share/elasticsearch/data
+    ports:
+      - 9209:9200
```

Comment:
Any reason for 9209?

Reply:
It is the port that is configured for default external elasticsearch port in dev-system. I use this port number to make it easy to run with dev-system where we only need to switch the elasticsearch to :external mode.